packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -0,0 +1,201 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const error = require('../error');
+const isA = require('joi');
+const ScopeSet = require('fxa-shared').oauth.scopes;
+const validators = require('./validators');
+
+module.exports = (log, db, config, customs, push, oauthdb, subhub) => {
+  const SUBSCRIPTIONS_MANAGEMENT_SCOPE =
+    'https://identity.mozilla.com/apps/subscriptions';
+
+  async function handleAuth(auth) {
+    const scope = ScopeSet.fromArray(auth.credentials.scope);
+    if (! scope.contains(SUBSCRIPTIONS_MANAGEMENT_SCOPE)) {
+      throw error.invalidToken('Invalid authentication scope in token');
```

Comment:
@rfk can probably chip in here on the semantics of the errors, I suspect a better error is [invalidScopes](https://github.com/lmorchard/fxa/blob/96a000ea6e702e7474e7b016e692b19cc7db9d0d/packages/fxa-auth-server/lib/error.js#L1010).

Reply:
Just went ahead and changed this to invalidScopes.